examples/common/10-deployment-kube2iam.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2017 the Heptio Ark contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  namespace: heptio-ark-server
+  name: ark
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        component: ark
+      annotations:
+        iam.amazonaws.com/role: arn:aws:iam::<AWS_ACCOUNT_ID>:role/<HEPTIO_ARK_ROLE_NAME>
+    spec:
+      restartPolicy: Always
+      serviceAccountName: ark
+      containers:
+        - name: ark
+          image: gcr.io/heptio-images/ark:latest
```

Comment:
Based on the file name, are you intending to deploy the Kube2iam daemonset here?

Reply:
No, this file does not attempt to install kube2iam, and I don’t think it should. It can be used by so many different apps, that it should be installed manually up front I think.

One potential option would be to make kube2iam an optional dependency of Ark in the Ark helm package, when this will be available. Then Helm could install kube2iam if user decides that they want to use it, and it is not yet available in the cluster.